types/ember/index.d.ts

```diff
@@ -1538,6 +1684,7 @@ namespace Ember {
         */
         has(name: string): boolean;
     }
+    const Route: Mixin<Route>;
```

Comment:
`Route` should still be a class, shouldn't it?

Reply:
Sorry - didn't mean to include that, though I think I'll need to make some changes.